specifyweb/specify/management/commands/run_key_migration_functions.py

```
@@ -0,0 +1,176 @@
+            "fix_schema_config": fix_schema_config,
+            "fix_tectonic_ranks": fix_tectonic_ranks,
+            "fix_misc": fix_misc,
+        }
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "functions",
+            nargs="*",
+            type=str,
+            choices=tuple(self.funcs.keys()),
+            help=f"Optional: specify one or more functions to run",
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)